players/g4_player.py

```diff
@@ -127,13 +136,22 @@ def _get_target_cells(self, size: int, cog: cell, xmax: int) -> list[cell]:
         arm_bottom = (y_cog - 3 * math.ceil((arm_cells_cnt - 1) / 2))
         arm_cell_ys = np.linspace(arm_bottom, arm_top, arm_cells_cnt, True, dtype=int)
 
+        # TODO: statically using xmax=51 can form a haircomb when A=3
+        # actually using xmax, self._reshape returns moves that cause
+        # separation, need to debug why, might be worth writing helpers
+        # to plot the target_cells, retract_cells, and to_occupy_cells
+        #
+        # Note that by using xmax as the x-value of wall cells, and
+        # xmax + 1 as the x-value of arm_cells will automatically cause
+        # the ameoba to shift right, but unsure why it's not working right
+        # now :(
         wall_cells = [
-            ( (xmax - 1) % 100, y % 100 )
+            ( xmax % 100, y % 100 )
             for y in wall_cell_ys
         ]
 
         arm_cells = [
-            ( xmax % 100, y % 100 )
+            ( (xmax + 1) % 100, y % 100 )
             for y in arm_cell_ys
         ]
```

Comment:
trying to make the ameoba move, but not working right now :(
simulator says that our moves are valid but causes separation.

Planning to write some helpers to visualize the `target_cells`, `retract_cells` and `to_occupy_cells` to help with debugging : )

Reply:
found why 😌 
<img width="1334" alt="image" src="https://user-images.githubusercontent.com/25857014/203226944-de5e6ab1-e106-407d-90e1-407935cd2cd6.png">